src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodcount/Example1.java

```diff
@@ -0,0 +1,31 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodCount"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodcount;
+
+// xdoc section -- start
+class Example1 {
+
+  public void outerMethod1(int i){}
+  public void outerMethod2(){}
+  public void outerMethod3(String str){}
+
+  private void outerMethod4(){
+    Runnable r = (new Runnable() {
+      public void run() {} // NOT counted towards ExampleClass
```

Comment:
towards Example1


Fix in each example.

Reply:
Done